tests/unit_tests/test_log.py

```diff
@@ -1,11 +1,19 @@
 import logging
+from functools import partial
 from logging import LogRecord
 from pathlib import Path
 from unittest.mock import MagicMock, patch
 
 import pytest
 
 from dodal import log
+from dodal.log import EnhancedRollingFileHandler, GELFTCPHandler
+
+
+def get_mock_streamhandler(level=20):
```

Comment:
Nit: Feels cleaner to use the enums rather than the numbers for the levels in this file. They're only tests though so meh